sdk/tests/events/changelog.test.ts

```
@@ -1,100 +1,116 @@
-import { AnchorProvider } from '@coral-xyz/anchor';
-import NodeWallet from '@coral-xyz/anchor/dist/cjs/nodewallet';
-import { bs58 } from '@coral-xyz/anchor/dist/cjs/utils/bytes';
-import { Connection, Keypair, PublicKey } from '@solana/web3.js';
-import { BN } from 'bn.js';
-import * as crypto from 'crypto';
+import { AnchorProvider } from "@coral-xyz/anchor";
+import NodeWallet from "@coral-xyz/anchor/dist/cjs/nodewallet";
+import { bs58 } from "@coral-xyz/anchor/dist/cjs/utils/bytes";
+import { Connection, Keypair, PublicKey } from "@solana/web3.js";
+import { BN } from "bn.js";
+import * as crypto from "crypto";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)